src.test/store-apply-collection.js

```diff
@@ -94,5 +94,12 @@ describe ('Store', () => {
       expect (store.select ('root.10').get ('value')).to.deep.equal ({year: 2016, name: 'foo'});
       expect (store.select ('root.13').get ('value')).to.equal ('none');
     });
+
+    it ('handle empty array collections', () => {
+      const store = Store.create ();
```

Comment:
@SL you should create a more meaningful state in the store. Expecting a non-existant `root` to be removed does not verify that the node gets removed, rather that it does not get created...